src/test/javascript/spec/component/course/course-management-exercises.component.spec.ts

```diff
@@ -64,22 +60,34 @@ describe('Course Management Exercises Component', () => {
         fixture = TestBed.createComponent(CourseManagementExercisesComponent);
         comp = fixture.componentInstance;
         courseService = TestBed.inject(CourseManagementService);
-        findStub = sinon.stub(courseService, 'find');
-        findStub.returns(of(new HttpResponse({ body: course })));
+        findStub = jest.spyOn(courseService, 'find');
+        findStub.mockReturnValue(of(new HttpResponse({ body: course })));
     });
 
     afterEach(function () {
-        sinon.restore();
+        jest.restoreAllMocks();
     });
 
     it('should initialize', () => {
         fixture.detectChanges();
-        expect(CourseManagementExercisesComponent).to.be.ok;
+        expect(CourseManagementExercisesComponent).toBeTruthy();
```

Comment:
Don't use such statements (Coding Guidelines).
Additionally, this test does nothing since the component will never be undefined. Either remove the test or test for specific attributes you want the component to be initialised with.

Reply:
This test was already there. I only changed it from Sinon to Jest.
But I agree with you that this test does not contribute to anything, so I will remove it.